### ISSUE TITLE
OS pop-up during world load

### ISSUE THREAD

**Issue by hfab** [lucidobservor/floramancer-modpack#6]
When I create or open a singleplayer world, a pop-up appears asking me to insert a disk into my DVD drive. Apparently the modpack tries to open a file on the D: disk. What is it trying to read and why? It should not access anything outside of the minecraft instance.
Most annoying is that in full screen mode, the popup is behind the minecraft window and cannot be accessed to close it. The only way to continue is to kill the java process.; switching to the pop-up window with alt-tab does not work and it is also not possible to switch to windowed mode with F11.
In windowed mode the pop-up also appears behind the game window, but now the game can be moved to make the pop-up visible.

OS: Windows 10 64-bit
Modpack version 1.2.1
Java version: 1.8.0_201
Launcher: multimc
Log starting at world load. At the end of this log the pop-up window is present and minecraft is waiting for it to be closed.
[worldload.log](https://github.com/lucidobservor/floramancer-modpack/files/2788799/worldload.log)
Pop-up window:
![image](https://user-images.githubusercontent.com/31726022/51630895-8d96f680-1f4b-11e9-982e-42114080b0aa.png)

**Comment by lucidobservor** [comment]
Well, points for having the strangest issue I've heard of. My best guess is that something went wrong with however you installed Minecraft/MultiMC or imported the pack into it. 

Things you can try:

- Attempt to run any other pack with your MultiMC setup and see if you get the same issue
- Disable the mod "Fullscreen Windowed (Borderless)" which is probably what's causing different fullscreen behavior than you expect
- Use the Windows 10 Task View to move the pop-up to a new virtual desktop so you can interact with it, then click Cancel to see what happens
- Run the pack using the Twitch launcher instead

Things you'll need to provide if you're still having problems:

- Use [pastebin](https://pastebin.com/), gist, or a similar service to share your logs instead of uploading the file. I've been told that MultiMC has an Upload button that directly uploads the log to paste.ee.
- Attach the full log, including what happens after you click cancel
- The results of taking the actions suggested above

Let me know if that helps resolve your issue.

**Comment by hfab** [comment]
I have not seen this issue with any other modpack, and I have played quite a few of them. Disabling the mod "Fullscreen Windowed (Borderless)" does not have any effect. Moving the pop-up to another virtual desktop makes the game window go to "Java(TM) Platform SE binary is not responding", blocking the use of the cancel button. I can dismiss the "not responding" pop-up, but it reappears as soon as I press the windows key to switch to the other desktop.
The same thing happens with the twitch launcher, but only when I change the java version to 1.8.0_201 instead of 1.8.0.51 that it uses by default. So it has something to do with the latest java release. Using the old java is not really an option since several mods need a newer version than 1.8.0_51.
I checked several other modpacks to see if any of them exhibit the same issue with the new java version, but none of them did (Stoneblock 2, Ultimate Alchemy Exam, Project Ozone 2, Pyramid Reborn 3.0, and Vanilla 1.12.2).
Fortunately I can play Floramancer by switching to windowed mode before loading the world. Then I can move the game window to get to the pop-up. After pressing cancel twice, the world loads in.
Full log is at https://paste.ee/p/lh7Kc

**Comment by hfab** [comment]
I have tracked the issue down to Antique Atlas.
When I disable Antique Atlas the problem goes away. When I disable everything but Antique Atlas the problem is still there. Updating to the latest Antique Atlas version does not solve it.
I have created a new issue at the Antique Atlas github [https://github.com/AntiqueAtlasTeam/AntiqueAtlas/issues/213](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/issues/213)

Edit: fixed link
